demos/python_demos/text_spotting_demo/README.md

```diff
@@ -116,13 +115,13 @@ To run the demo, you can use public or pre-trained models. To download the pre-t
 
 > **NOTE**: Before running the demo with a trained model, make sure the model is converted to the Inference Engine format (`*.xml` + `*.bin`) using the [Model Optimizer tool](https://docs.openvinotoolkit.org/latest/_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide.html).
 
-To run the demo, please provide paths to the model in the IR format and to an input video, image, or folder with images:
+To run the demo, please provide paths to the model in the IR format and to an input with images:
```

Comment:
is that demo accept only video (from file or from camera) but not image?

Reply:
Depends on having ffmpeg plugin for opencv on Windows. So my change is to avoid specifying what sources are supported because it is up to opencv